roadmap.asciidoc

```diff
@@ -13,23 +13,22 @@ but this page is a good starting point to see where we are going.
 This roadmap is subject to changes.
 Please get https://groups.google.com/forum/#!forum/debezium[in touch] if you think anything important is missing on the roadmap.
 
-== 1.0
+== 1.1
 
-* Revise and align configuration options, e.g. snapshot modes
 * Provide a Quarkus extension facilitating the application-side part when implementing the outbox pattern
 * Incubating support for IBM DB2
-* Unify more parts of the code base across connectors, e.g. handling of whitelists
-
-== 1.1
-
 * Expose topic with transaction events (begin, end)
+* Incubating support for CloudEvents
 * Analyze and fix performance bottlenecks
-* Relocate Debezium's Java packages to `io.debezium.internal.*`, making it apparent that there's no public Java API/SPI (apart from the embedded connector); we may add such API/SPI later on, allowing to implement custom connectors on the foundations of Debezium, but it won't be supported as of 1.0
 * Rebase MySQL and MongoDB connectors to be based on new common framework developed during the work on the Oracle and SQL Server connectors
```

Comment:
@gunnarmorling Maybe this one should go to 1.x too?

Reply:
Yes indeed. I'll move it and merge. Thanks!